src/components/plan/PlanForm.svelte

```diff
@@ -131,8 +135,7 @@
             options={tags}
             disabled={!hasPermission}
```

Comment:
Rather than disabling it when there's no permission, would it be possible to instead define a `editable` field? That way the hover can still show the permission error. I think having the `disabled` prop makes the `permissionHandler` effectively moot. If it's not possible, then maybe consider removing the `permissionHandler` since i'm not sure it does anything.

Reply:
Implemented, want to give it a quick spin?